luminance-gl/src/gl33/pipeline.rs

```diff
@@ -17,6 +17,7 @@ use std::cell::RefCell;
 use std::marker::PhantomData;
 use std::rc::Rc;
 
+use super::state::ScissorState;
```

Comment:
Please use `crate::…` instead of `super::…`. I know it might be longer, but it’s coherent with the rest of the codebase. :)

Reply:
Ah, whoops, rust-analyzer auto-import did that :)